config.yml.example

```
@@ -97,3 +97,15 @@ jail:
     link_influxdb: influxdb
     influxdb_password: unifi-poller
     poller_password: upoller
+
+  grafana:
+    blueprint: grafana
+    ip4_addr: 192.168.1.100/24
+    gateway: 192.168.1.1
+    password: grafana
+    link_influxdb: influxdb
+    database:
+      name: Unifi Controller
+      database: unifi
+      user: unifi
+      password: unifi-poller
```

I'm afrait people are going to mess up another indentation.
I also try to prevent myself from adding to many indentations...

You could consider doing just:
```suggestion
    database_name: Unifi Controller
    database_database: unifi
    database_user: unifi
    database_password: unifi-poller
```

---

To me it makes more sense with the indentation, but I totally understand that it can (and will) get easily messed up. Since the intention is less indentation, I'll change it. Should be less confusing for others as well.